### PR TITLE
New first branch

### PR DESCRIPTION
**Pull request by thapdigov**
I have made changes in projects which you said last time

### CODE REVIEW

File: src/main/java/az/edu/turing/modul1/Task1A.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            } else {
+                System.out.println("Your entered number is large,please ,try again");
+                number = sc.nextInt();
+            }
+            numbers[count] = number;
+            count++;
+
+
+        }
+        int finalArrays[] = Arrays.copyOf(numbers, count);
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/turing/modul1/Task1A.java:48]: please follow java convention `int[] finalArrays`

File: src/main/java/az/edu/turing/modul1/Task1A.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        System.out.print("Enter the name: ");
+        String name = sc.nextLine();
+
+        System.out.println("Enter the number you want to,between 0 and 100");
+        Integer number = sc.nextInt();
+
+        int[] numbers = new int[100];
+        int count = 0;
+        numbers[count] = number;
+        count++;
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/turing/modul1/Task1A.java:29]: instead you can write `numbers[count++] = number;`
also you write same code in 43,44 lines. Then we should avoid this dublication

File: src/main/java/az/edu/turing/modul1/Task2B.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+import java.util.Scanner;
+
+public class Task2B {
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("---------Welcome to the Week's planner----------");
+        String[][] weeks = new String[7][2];
+        weeks[0][0] = "Sunday";
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/turing/modul1/Task2B.java:11]: you can move this initialization to another method

File: src/main/java/az/edu/turing/modul1/Task2B.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+                    2.Change plan of the day
+                    3.Exit""");
+            int menu = scanner.nextInt();
+            scanner.nextLine();
+            switch (menu) {
+                case 1: {
+                    System.out.println("Enter the number of  day  in week:");
+                    int numberDay = scanner.nextInt();
+                    while (numberDay < 0 || numberDay > 6) {
+                        System.out.println("Please,Number has been between 0 and 6");
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/turing/modul1/Task2B.java:39]: get input between 1-7 from user, and handle it your side

File: src/main/java/az/edu/turing/modul1/Task2B.java [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+        weeks[4][0] = "Thursday";
+        weeks[4][1] = "have a fun with friends";
+        weeks[5][0] = "Friday";
+        weeks[5][1] = "go to the fitness";
+        weeks[6][0] = "Saturday";
+        weeks[6][1] = "run in the morning!";
+
+        while (true) {
+            System.out.println("""
+                    Choose the Menu:
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/turing/modul1/Task2B.java:28]: the condition of the problem is not like that, we just receive an input and therefore we already choose an action

File: src/main/java/az/edu/turing/modul1/Task3C.java [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+            for (int i = 0; i < board.length; i++) {
+                for (int j = 0; j < board[i].length; j++) {
+                    System.out.print(board[i][j]);
+                }
+                System.out.println();
+            }
+
+            System.out.print("Enter the row (1-5): ");
+            int row = sc.nextInt() - 1;
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/turing/modul1/Task3C.java:36]: move the getValidİnput to another method

File: src/main/java/az/edu/turing/modul1/Task3C.java [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+        int count = 0;
+
+        for (int i = 0; i < board.length; i++) {
+            Arrays.fill(board[i], " - ");
+
+        }
+
+        while (true) {
+
+            for (int i = 0; i < board.length; i++) {
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/turing/modul1/Task3C.java:28]: move this print to another method

File: src/main/java/az/edu/turing/modul1/Task3C.java [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+        String[][] board = new String[5][5];
+        Random random = new Random();
+
+        int targetRow = random.nextInt(5);
+        int targetColumn = random.nextInt(5);
+
+        System.out.println("All set. Get ready to rumble!");
+        int count = 0;
+
+        for (int i = 0; i < board.length; i++) {
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/turing/modul1/Task3C.java:21]: move this initialization to another method

**Review by sabuhihuseynov — CHANGES_REQUESTED** [review]
pls look

File: src/main/java/az/edu/turing/modul1/Task1A.java (hunk 2 above)
Comment (thapdigov) [reply src/main/java/az/edu/turing/modul1/Task1A.java:29]: if I write  as you say, the first element will not be  add to finalArrays

File: src/main/java/az/edu/turing/modul1/Task2B.java (hunk 5 above)
Comment (thapdigov) [reply src/main/java/az/edu/turing/modul1/Task2B.java:28]: I looked at the children's projects, most of them did not write the function of changing the task of a certain day, but in the condition of the task, there was also a change of the task of the day.

File: src/main/java/az/edu/turing/main/App.java [hunk 9]
```diff
@@ -0,0 +1,13 @@
+package az.edu.turing.main;
+
+/**
+ * Hello world!
+ *
+ */
+public class App 
+{
```
Comment (sabuhihuseynov) [inline src/main/java/az/edu/turing/main/App.java:8]: fix this) also take attention branch name

File: src/main/java/az/edu/turing/modul1/Task1A.java (hunk 2 above)
Comment (sabuhihuseynov) [reply src/main/java/az/edu/turing/modul1/Task1A.java:29]: you can implement logic for prevent that, pls think about it, also move input validation to another method

File: src/main/java/az/edu/turing/modul1/Task2B.java (hunk 5 above)
Comment (sabuhihuseynov) [reply src/main/java/az/edu/turing/modul1/Task2B.java:28]: I agree with you, but in condition don't say that, show user menu and then do action for that, in condition said that wait input from user (as a String) and do action based on this value

**Review by sabuhihuseynov — CHANGES_REQUESTED** [review]
pls look

**Pull request closed without merging**